clients/unshaded/src/main/java/tachyon/client/file/options/CreateOptions.java

```diff
@@ -143,6 +143,15 @@ public UnderStorageType getUnderStorageType() {
   }
 
   /**
+   * @return the name : value pairs for all the fields
+   */
+  @Override
+  public String toString() {
+    return "CreateTOptions{mBlockSizeBytes=" + mBlockSizeBytes + ",mRecursive=" + mRecursive
```

Comment:
looks like toStringHelper will insert a space between fields (like  
```
// Returns "ClassName{x=1, y=foo}"
```
), which will be a problem unless we rewrite the tests